src/app/modules/notification/services/notification.service.ts

```diff
@@ -0,0 +1,130 @@
+import { Injectable } from '@angular/core';
+
+export interface Notification {
```

Comment:
I prefer to have every type exported in a @type folder, it's easier to find them.

In your case the folder will be placed at the same level than `/services`